src/katt_callbacks.erl

```diff
@@ -241,7 +245,30 @@ http_request( #katt_request{ method = Method
   Hdrs1 = proplists:delete("x-katt-sleep", Hdrs0),
   Hdrs = proplists:delete("x-katt-timeout", Hdrs1),
   timer:sleep(Sleep),
-  lhttpc:request(Url, Method, Hdrs, Body, Timeout, []).
+  %% need to module-prefixed call for meck's case
+  %% http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+  katt_callbacks:http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+
+http_request_now(Url, Method, Hdrs, Body, Timeout, []) ->
```

Comment:
> Re: util - I don't see a reason why a wrapper around an http client shouldn't be in some *_util, 

because it doesn't have any utility outside `katt.erl`,.. or to put it differently - if it wasn't for lhttpc-> hackney, for meck, and then for the required module-prefixed call, would you have looked at the code and though "hmm, this http_request wrapper around lhttp:request" should probably go out of katt.erl into katt_util.erl ?

re: tests - #22 :p Nothing wrong, but it's not easy(tm) to just add a simple test. Reminds me of yaws testing :)